src/Main.java

```diff
@@ -178,11 +196,20 @@ else if (input.equals("Make order")){
                             System.out.println("Choose your payment:\n\tFor ImmediatePayment press '1'\n\tFor DelayedPayment press '2'\n\t" +
                                     "Press '3' when you're done");
                             paymentType = myObj.nextLine();
-                            if (paymentType.equals("3")) continue;
+                            if (paymentType.equals("3")) {
+                                if(o.getTotal() > o.getPaid()){
+                                    System.out.println("Reminder: You didn't finish paying for this order.");
+                                }
+                                break;
```

Comment:
sure that break is right here? not continue?

Reply:
If you put 'continue' instead of the 'break' means you're inside the if statement(paymentType=3) if you continue it will go to the next iteration and then the if statement would be FALSE. Instead, you hit break and you just don't check it anymore(because it's already 3) what makes it a little bit faster ( one check less)